docker/Dockerfile

```diff
@@ -22,22 +22,19 @@ RUN pacman -Syyuu --noconfirm --needed --overwrite \*
 # Add a script that installs packages
 COPY usr/local/bin/install-package /usr/local/bin/install-package
 
-# Install latest archlinux-keyring package so we have the most up-to-date keys
-RUN install-package wget
-RUN wget https://www.archlinux.org/packages/core/any/archlinux-keyring/download/ -O /var/tmp/archlinux-keyring.pkg.tar.zst
-RUN pacman -U /var/tmp/archlinux-keyring.pkg.tar.zst --noconfirm
-
 # Install base packages needed for building general toolchain
 # If you have a tool that's needed for a specific module install it before that module
 RUN install-package \
+    wget \
     sudo \
     python \
     python-pip \
     base-devel \
     ninja \
     cmake \
     meson \
-    git
+    git \
+    ncurses
```

Comment:
Why do we need to install ncurses here?

Reply:
I added that because [arch has it listed as a dependency of intel-graphics-compiler](https://archlinux.org/packages/community/x86_64/intel-graphics-compiler/)

Reply:
If it's a dependency then we should build it ourselves. Anything that'll actually run on the robot should be built by us.